lib/jekyll/configuration.rb

```diff
@@ -226,9 +226,10 @@ def backwards_compatibilize
       # Provide backwards-compatibility
       check_auto(config)
       check_server(config)
+      check_plugins(config)
 
       renamed_key "server_port", "port", config
-      renamed_key "plugins", "plugins_dir", config
+      renamed_key "plugins", "gems", config, false
```

Comment:
It was intentional 😊  This way we can still use `config.gems` internally but i can change this of course